cmd/shfmt/main_test.go

```diff
@@ -70,9 +71,9 @@ var walkTests = []struct {
 	{None, false, "ext.other", " foo"},
 	{None, false, "ext-shebang.other", "#!/bin/sh\n foo"},
 	{None, false, "shebang-nospace", "#!/bin/envsh\n foo"},
-	{None, false, filepath.Join(".git", "ext.sh"), " foo"},
-	{None, false, filepath.Join(".svn", "ext.sh"), " foo"},
-	{None, false, filepath.Join(".hg", "ext.sh"), " foo"},
+	{Skip, false, filepath.Join("foo", ".git", "ext.sh"), " foo"},
```

Comment:
why the change with `"foo"`? that seems unnecessary

Reply:
That was just by mistake, removed now.